### PR TITLE
#164005328 signup ui page 

### PR DESCRIPTION
**Pull request by Nziranziza**
#### What does this PR do?
add signup html page
#### Description of Task to be completed?
- write HTML
- add CSS
- add reponsiveness
#### How should this be manually tested?
- git clone 
- npm install
- npm run build-scss
- open .ui/pages/signup.html
#### Any background context you want to provide?
- [Desktop version mockup](https://drive.google.com/drive/folders/1lcFZ3-tOAoN8g5_e6ORaKs8jEVI783jg)
- [Mobile and Tablet version mockup](https://drive.google.com/drive/folders/1chupPda0Ny-iePagVr0_QCZgEhBabhM_)
#### What are the relevant pivotal tracker stories?
#164005328
###### Desktop version
screen shot 2019-02-22 at 19 40 23

###### Mobile and Tablet version
screen shot 2019-02-22 at 19 42 38

### CODE REVIEW

**Comment by Nziranziza** [comment]
@mcaleb808 @filschristian

File: ui/scss/_input.scss [hunk 1]
```diff
@@ -36,4 +36,4 @@ input::placeholder {
 }
 .input.primary input::placeholder {
   color: $primary;
-}
```
Comment (mcaleb808) [inline ui/scss/_input.scss]: Try to fix this issue!

**Review by mcaleb808 — APPROVED** [review]

**Review by filschristian — APPROVED** [review]
@Nziranziza Why did you remove prettier in .eslintrc file???? 
@Nziranziza please change the title of the PR!!! follow best practices

**Review by filschristian — COMMENTED** [review]
Work on my reviews

**Pull request merged**